test/backend.js

```diff
@@ -226,10 +226,10 @@ describe('Full walkthrough', function () {
 
     it("Should invite Bob to Alice's group", function (done) {
       var mailbox = users.bob.mailbox
-      var invite = new Events.PostInvite({groupId: groups.group1.toHash()}, latestHash(mailbox))
+      var invite = new Events.PostMessage({messageType: 'Invite', message: groups.group1.toHash()}, latestHash(mailbox))
```

Comment:
`messageType: 'Invite'` <- This is going to cause difficulties maintaining the codebase down the line when there are a ton of message types.

Anytime an enum like this is used, it's best to use a constant enum-type.

So add to the `PostMessage` class, as static consts, the types. E.g. `static const TypeInvite = 'invite'`, and use as `PostInvite.TypeInvite`.